app/Http/Controllers/Admin/OrderController.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\Order;
+use App\Models\OrderItem;
+use App\Models\Ticket;
+use Illuminate\Http\Request;
+use Illuminate\Validation\ValidationException;
+
+class OrderController extends Controller
+{
+    public function index(Request $request)
+    {
+        $query = Order::with(['user', 'orderItems', 'refund']);
+
+        if ($request->filled('status')) {
+            $query->where('status', $request->input('status'));
+        }
+
+        if ($request->filled('user')) {
+            $query->whereHas('user', function ($q) use ($request) {
+                $q->whereRaw('LOWER(name) LIKE ?', ['%' . strtolower($request->input('user')) . '%']);
+            });
+        }
+
+        if ($request->filled('from_date')) {
+            $query->whereDate('created_at', '>=', $request->input('from_date'));
+        }
+
+        if ($request->filled('to_date')) {
+            $query->whereDate('created_at', '<=', $request->input('to_date'));
+        }
+
+        $orders = $query->latest()->paginate(10);
+        $statuses = Order::select('status')->distinct()->pluck('status');
+
+        return view('pages.admin.orders.index', compact('orders', 'statuses'));
+    }
+
+    public function create()
+    {
+        $query = Ticket::with('event');
+
+        if (request('search')) {
+            $search = strtolower(request('search'));
+            $query->whereHas('event', fn($q) => $q->whereRaw('LOWER(title) LIKE ?', ["%{$search}%"]))
+                  ->orWhereRaw('LOWER(category) LIKE ?', ["%{$search}%"]);
+        }
+
+        if (request('category')) {
+            $query->where('category', request('category'));
+        }
+
+        $tickets = $query->where('quantity', '>', 0)->get();
+
+        return view('pages.admin.orders.create', compact('tickets'));
+    }
+
+    public function store(Request $request)
+    {
+        $validated = $request->validate([
+            'user_id' => 'required|exists:users,id',
+            'status' => 'required|string',
```

Comment:
Pole status nie ma walidacji wartości – każda dowolna stringowa wartość przejdzie.
```suggestion
            'status' => 'required|string|in:pending,paid,cancelled,refunded',
```